src/repository/ItemRepository.java

```diff
@@ -0,0 +1,27 @@
+package repository;
+
+import model.Item;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ItemRepository {
+    private Map<String, Item> storage = new HashMap<>();
```

Comment:
This should be part of the constructor and name should be items instead of storage

You don't want people to be confused and understand too many things 

Based on your code I would have to understand what storage